packages/gatsby-cli/src/reporter/index.js

```diff
@@ -1,17 +1,15 @@
 // @flow
-
-const { createReporter } = require(`yurnalist`)
+// loads Object.entries polyfill on node 6
+// @see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/entries
+require(`object.entries/auto`)
```

Comment:
I think this should be handled by https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-cli/src/index.js#L5 ? We do use `object.entries` in different parts of the code and it works, and it's most often just tests that fail on those (because they don't use `gatsby-cli` as entry point?

Reply:
I see, should I just put it in the failing tests? Or not bother with node 6 at all?